feature/session/src/main/java/io/github/droidkaigi/confsched2019/session/ui/item/EqualableContentsProvider.kt

```diff
@@ -0,0 +1,27 @@
+package io.github.droidkaigi.confsched2019.session.ui.item
+
+import java.util.Arrays
+
+interface EqualableContentsProvider {
```

Comment:
Can you move this interface to androidcomponent module?

Reply:
sure, item folder would be good?